### PR TITLE
Initial implementation and tests

### PR DESCRIPTION
**Pull request by honzajavorek**
- Moved all the relevant code and tests from Fury.js.
- Employed `coffeelint` so the code is readable at least when readers are half-drunk, not only if readers are completely poisoned and while hallucinating, they are coincidentally recognizing where function calls start and where they finish.
- Not tested on anything higher than Node 0.10 as `robotskirt` doesn't support anything newer.
- Removed `underscore` dependency.
- There's browser integration for tests ready, but due to `robotskirt` it's temporarily disabled.

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+// See https://github.com/1hella/eslint-config-airbnb-es5
+{
+  "extends": "eslint-config-airbnb-es5"
```
Comment (danielgtaylor) [inline .eslintrc]: It looks like this is all CoffeeScript. Any reason for this file?

File: README.md [hunk 2]
```diff
@@ -38,6 +38,44 @@ To be able to work with both of these formats through some sort of uniform inter
...
+var adapter = metamorphoses.createAdapter(mimeType);
+
+
+// You can also import the adapter directly:
+// -   API Blueprint AST: apiBlueprintAdapter
+// -   Legacy Apiary Blueprint AST: apiaryBlueprintAdapter
+var adapter = metamorphoses.apiBlueprintAdapter;
+
+
+parse(source, ..., function (err, result) {
```
Comment (danielgtaylor) [inline README.md:67]: Not really clear where this `parse` method is coming from. I'm guessing `protagonist`.

File: .eslintrc (hunk 1 above)
Comment (honzajavorek) [inline .eslintrc]: leftover! thanks for catching this

**Comment by danielgtaylor** [comment]
Looking good! :+1:

File: README.md (hunk 2 above)
Comment (honzajavorek) [inline README.md:67]: It is pseudo-code parse which can come from protagonist or some sort of wrapper around [blueprint-parser](https://github.com/apiaryio/blueprint-parser). I should be more clear with this.

**Comment by honzajavorek** [comment]
@danielgtaylor addressed your comments. Feel free to assign yourself and apply green label if you are satisfied (Tony has holiday).

**Comment by danielgtaylor** [comment]
:shipit:

**Comment by honzajavorek** [comment]
Thanks!

**Pull request merged**